topics/proteomics/tutorials/peptide-library-data-analysis/tutorial.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: tutorial_hands_on
+
+title: Peptide Library Data Analysis
+level: Intermediate
+questions:
+  - How to utilize quantitative properties of amino acids and peptide sequence to analyse peptide data?
+objectives:
+  - Calculate descriptors
+  - Quantitative analysis of peptide sequence properties
+time_estimation: '20m'
+contributors:
+   - jaidevjoshi83
+   - blankenberg
+---
+
+# Introduction
+{:.no_toc}
+
+
+Several computational methods have been proven very useful in the initial screening and prediction of peptides for various biological properties. These methods have emerged as effective alternatives to the lengthy and expensive traditional experimental approaches. In this tutorial, we will be discussing how to analyze peptide libraries on the basis of quantitative properties. In this tutorial, we will learn how to use different utilities of PDAUG to calculate various peptide-based features and utilize these features for various informative plots.
+
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+## Peptide Data
+
+In this step, we will retrieve the inbuild dataset, which contains anti-microbial (AMPs) and transmembrane peptides (TMPs).
+
+
+> ### {% icon hands_on %} Hands-on: Fetching inbuild data
+> 
+> 1. {% tool [PDAUG Peptide Data Access](toolshed.g2.bx.psu.edu/repos/jay/pdaug_peptide_data_access/pdaug_peptide_data_access/0.1.0) %} with the following parameters:
+>    - *"Datasets"*: `AMPvsTM` 
+>
+{: .hands_on}
+
+
+## Converting tabular data into fasta format
+
+In this step, we will be converting and splitting the tabular data file into fasta format. This tool splits data into two files based on their class labels.
+
+> ### {% icon hands_on %} Hands-on: Converting tabular data into fasta formate
+> 
+> 1. {% tool [PDAUG TSVtoFASTA](toolshed.g2.bx.psu.edu/repos/jay/pdaug_tsvtofasta/pdaug_tsvtofasta/0.1.0) %} with the following parameters:
+>    - {% icon param-file %} *"Input file"*: `PDAUG Peptide Data Access - AMPvsTM (tabular)` (output of **PDAUG Peptide Data Access** {% icon tool %})
+>    - *"Data conversion"*: ` WithClassLabel `
+>
```

Comment:
It is not clear to me why I have to split my dataset in the first place. Then it is confusing that it says split according to “class labels”, which seems to correspond the “targets” column. Consistent nomenclature or better explanation about what is done why would help here. 
Regarding the output of the tool, I am wondering how I know which of the two outputs is AMP and which TM?

Reply:
-`PDAUG Peptide Data Access` tool returns data as a tabular file that contains sequences from both the classes. In order to utilize this data in the next steps first, we need to convert tabular data into fasta format. If data contains sequences from two different classes `PDAUG TSVtoFASTA` tool converts and splits data into two separate files for each of the class,  AMPs, and TMPs.  The reason behind converting and splitting the data is that all the downstream tools require two separate files if we are comparing two different peptide classes or calculating features.
- tools and workflow have been updated "target" column name replaced with "class_label" and peptides have been labeled with their actual classes (TMPs or AMPs) 
- PDAUG TSVtoFASTA tool modified and tag with each of the peptide sequences name has been added in the fast file for example ">Peptide_1_TMPs".  Additionally, this tool splits data as per the occurrence of the classes in a tabular data file which returns by the PDAUG Peptide Data Access tool.